### PR TITLE
Document upstream cookies

### PR DESCRIPTION
**Pull request by dasrecht**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
     * [Environment type](architecture/environment_type.md)
     * [E-Mail Delivery](architecture/e-mail_delivery.md)
     * [URL Structure](architecture/url-structure.md)
+    * [Upstream Testing](architecture/backend_upstream.md)
```
Comment (Schnitzel) [inline SUMMARY.md]: I would call it "Varnish Testing" as nobody really knows what "Upstream" is

File: architecture/backend_upstream.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Backend and Upstream Testing
+
+Sometimes you need to debug an issue on production without varnish being in your way.
```
Comment (Schnitzel) [inline architecture/backend_upstream.md]: maybe explain shortly that production have varnish enabled and development (and also local development environemnts) have varnish around but disabled

**Review by Schnitzel — CHANGES_REQUESTED** [review]

File: SUMMARY.md (hunk 1 above)
Comment (dasrecht) [reply SUMMARY.md]: Done in newest commit

File: architecture/backend_upstream.md (hunk 2 above)
Comment (dasrecht) [reply architecture/backend_upstream.md]: Added a bit of text to reflect the behaviour of varnish on development and production :)

**Review by Schnitzel — APPROVED** [review]

**Pull request merged**